src/main/java/com/ericsson/vici/api/entities/Settings.java

```diff
@@ -17,16 +17,26 @@
 package com.ericsson.vici.api.entities;
 
 
-import java.util.Arrays;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import java.util.HashMap;
-import java.util.List;
 
 import static com.ericsson.vici.Fetcher.*;
 
+@Data
+@NoArgsConstructor
 public class Settings {
-    public static final String propertiesVersion = "2.2 - Added cut at events, changed standard values to fit new algorithm.";
 
-    private static final String[] eventTypesAggregatedOn = {
+    @JsonIgnore
+    private final String latestVersion = "2.3 - Eiffel-ER with Spring property as default.";
```

Comment:
Why is this needed or what is it used for?

Reply:
This is used for overriding old settings if a user pulls down a new version with different structured settings (most relevant during development before the structure of settings etc is more stable). The idea is to compare the stored settings whit the latest version to see if a migration of settings is required

Reply:
ok